### PR TITLE
Reorganize Method tests

### PR DESCRIPTION
**Pull request by aweng98**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by aweng98** [comment]
jenkins retest

File: automation/logback-test.xml [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
     <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
         <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>%-70(%d{HH:mm:ss} [%thread]) %highlight(%-5level) %-50([%-30.30logger{30}:%method]) - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss} [%-31.31(%thread)] %highlight(%-5level) [%-25.25logger{25}%-15(:%method)] - %msg%n</pattern>
```
Comment (aweng98) [inline automation/logback-test.xml:11]: new logback formatting

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala [hunk 2]
```diff
@@ -43,14 +43,9 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
       page.awaitReady() // check if click happened
     ) match {
       case Failure(e) => // click failed
-        // TODO remove after proving it worked
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala:46]: remove logs that were used for debugging flaky test method

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala [hunk 3]
```diff
@@ -107,6 +109,12 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
     modal.clickAbortSubmissionButton
   }
 
+  def getApiSubmissionStatus(billingProject: String, workspaceName: String, submissionId: String)(implicit token: AuthToken): String = {
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala:112]: use api to fetch workflow submission status. it helps tests not depend on UI all the time

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala [hunk 4]
```diff
@@ -21,34 +22,6 @@ class MethodLaunchSpec extends FreeSpec /*with ParallelTestExecution*/ with Matc
   val missingInputsErrorText: String = "is missing definitions for these inputs:"
   val testData = TestData()
 
-  "launch workflow and delete a workflow" in {
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala:24]: too many tests that launch workflow, causing long test execution time. `delete workflow` testing moved to `BillingSpec`

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala [hunk 5]
```diff
@@ -97,6 +97,12 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cl
               assert(submissionDetailsPage.verifyWorkflowSucceeded())
               //END: Test running anlysis in workspace
 
+              // BEGIN: should be able to delete workflow
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala]: delete workflow testing that were in `MethodLaunchSpec`

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala [hunk 6]
```diff
@@ -22,33 +21,6 @@ class DataSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpec w
   val methodConfigName: String = randomIdWithPrefix(SimpleMethodConfig.configName)
   val testData = TestData()
 
-  def makeMetadataFile(filePrefix: String, headers: List[String], rows: List[List[String]]): File = {
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala:25]: Workspace Data table metadata testing moved to new spec called `MetadataImportSpec`

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala [hunk 7]
```diff
@@ -207,164 +180,4 @@ class DataSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpec w
     }
   }
 
-  "Writer and reader should see new columns" - {
```
Comment (aweng98) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala:210]: moved to new `MetadataImportSpec`

File: automation/build.sbt [hunk 8]
```diff
@@ -58,7 +58,7 @@ Test / logBuffered := false
   *  Warning: can't set too high (set at 10 would crashes OS)
   *  This is not number of threads in each JVM. That would be up to sbt.
   */
-Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 4))
+Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 6))
```
Comment (andy7i) [inline automation/build.sbt]: Is this change related to reorganizing method tests?

Comment (aweng98) [reply automation/build.sbt]: yes and no. number of JVMs are upped to see whether there are any previously undetected Methods tests interference.

**Comment by aweng98** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala (hunk 5 above)
Comment (MatthewBemis) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala]: this deletes a method config, not a workflow

**Review by MatthewBemis — APPROVED** [review]
one note about a comment. not mandatory right now

**Comment by aweng98** [comment]
jenkins retest

**Review by gdlex4015 — APPROVED** [review]
